scale/ingest/test/test_views.py

```diff
@@ -1785,21 +953,26 @@ def test_invalid_configuration(self):
 
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST, response.content)
 
-    def test_successful(self):
+    @patch('queue.models.CommandMessageManager')
+    def test_successful(self, mock_msg_mgr):
         """Tests calling the create Strike view successfully."""
 
         json_data = {
             'title': 'Strike Title',
             'description': 'Strike description',
             'configuration': {
-                'version': '1.0',
-                'mount': 'host:/my/path',
-                'transfer_suffix': '_tmp',
+                'version': '2.0',
```

Comment:
Use v6 json? I think the only difference from 2 is the version.

Reply:
You are correct. Changed to version 6.